ngraph/core/include/ngraph/opsets/opset.hpp

```diff
@@ -132,4 +132,5 @@ namespace ngraph
     const NGRAPH_API OpSet& get_opset2();
     const NGRAPH_API OpSet& get_opset3();
     const NGRAPH_API OpSet& get_opset4();
+    const NGRAPH_API OpSet& get_opset5();
```

Comment:
You need to update the `inference-engine/src/readers/ir_reader/ie_ir_parser.cpp` line 489 with the value "5" instead of "4", otherwise the IR parser will not read opset5 ops.

Reply:
Also, please, add at least one test to the `tests/functional/inference_engine/ngraph_reader` for some of the XXXSequence op.

Reply:
`ie_ir_parser.cpp` has been updated.  Ngraph reader tests will be added as part of  35577, 39253, 35565 tickets (update IR Reader to support LSTM/GRU/RNN Sequences). We don't generate IR with sequences now.